### ISSUE TITLE
Error types should be only ETSI URIs

### ISSUE THREAD

**Issue by jmcanterafonseca** [Fiware/context.Orion-LD#22]
I have noticed that sometimes error payloads are generated as follows:

```
"type": "http://example.org/ngsi-ld/errors/ResourceNotFound" 
```

Error payloads "type" member shall always contain ETSI-defined URIs:

Error Type | HTTP status
-- | --
http://uri.etsi.org/ngsi-ld/errors/InvalidRequest | 400
http://uri.etsi.org/ngsi-ld/errors/BadRequestData | 400
http://uri.etsi.org/ngsi-ld/errors/AlreadyExists | 409
http://uri.etsi.org/ngsi-ld/errors/OperationNotSupported | 422
http://uri.etsi.org/ngsi-ld/errors/ResourceNotFound | 404
http://uri.etsi.org/ngsi-ld/errors/InternalError | 500
http://uri.etsi.org/ngsi-ld/errors/TooComplexQuery | 403
http://uri.etsi.org/ngsi-ld/errors/TooManyResults | 403

**Comment by kzangeli** [comment]
Yeah, not sometimes, more like always.
This is how the conversion to error string is implemented:
```
// -----------------------------------------------------------------------------
//
// errorTypeStringV -
//
static const char* errorTypeStringV[] =
{
  "http://example.org/ngsi-ld/errors/InvalidRequest",
  "http://example.org/ngsi-ld/errors/BadRequestData",
  "http://example.org/ngsi-ld/errors/AlreadyExists",
  "http://example.org/ngsi-ld/errors/OperationNotSupported",
  "http://example.org/ngsi-ld/errors/ResourceNotFound",
  "http://example.org/ngsi-ld/errors/InternalError"
};
```
Seems like there's been a change in the spec.
Not a problem, very easy to fix.

**Comment by kzangeli** [comment]
Fixed in ec8501e
